ic-utils/src/interfaces/management_canister.rs

```diff
@@ -194,6 +199,37 @@ impl std::fmt::Display for CanisterStatus {
     }
 }
 
+/// A log record of a canister.
+#[derive(Default, Clone, CandidType, Deserialize, Debug, PartialEq, Eq)]
+pub struct CanisterLogRecord {
+    /// The index of the log record.
+    pub idx: u64,
+    /// The timestamp of the log record.
+    pub timestamp_nanos: u64,
+    /// The content of the log record.
+    #[serde(with = "serde_bytes")]
+    pub content: Vec<u8>,
+}
+
+impl std::fmt::Display for CanisterLogRecord {
```

Comment:
The purpose of Display is to generate user-facing output; why is it being implemented in terms of Debug, which will just show programmer output (especially as a byte array)?

Reply:
yes, moving it to dfx side is a good idea, thank you.